Foundation/NSValue.swift

```diff
@@ -98,7 +98,7 @@ open class NSValue : NSObject, NSCopying, NSSecureCoding, NSCoding {
     }
     
     public convenience required init(bytes value: UnsafeRawPointer, objCType type: UnsafePointer<Int8>) {
-        if type(of: self) == NSValue.self {
+        if Swift.type(of: self) == NSValue.self {
```

Comment:
Hmmph that still seems like a bug to me... type as the variable seems quite clear that it is not invoked as a function especially since it has a named parameter. But ok, I guess chock it up to the same failure as print and NSView.print :/